appengine/appengine.bzl

```diff
@@ -155,6 +155,8 @@ def _war_impl(ctxt):
       appengine_sdk = f.path
```

Comment:
I think this can be changed to short_path to avoid the path munging on line 159.